### PR TITLE
Docker image and CI workflows

### PR DESCRIPTION
**Pull request by Zebedeusz**

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM pryv/base:1.5.1 as build-stage
```
Comment (IevaNavikiene) [inline build/Dockerfile:1]: why do you use pryv image and not something lighter like node:lts-alpine for building the first image?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+node_modules
```
Comment (IevaNavikiene) [inline .dockerignore:1]: this example https://github.com/ledermann/docker-vue/blob/master/.dockerignore 
shows additional 
.git
.dockerignore
.gitignore
README.md
Dockerfile

I also usually exclude tests

File: build/Dockerfile (hunk 1 above)
Comment (Zebedeusz) [reply build/Dockerfile:1]: I was told to do so by Ilia. That we should use this image in every case

File: .dockerignore (hunk 2 above)
Comment (Zebedeusz) [reply .dockerignore:1]: Done

**Pull request merged**